mms/model_loader.py

```diff
@@ -29,7 +29,7 @@
 MANIFEST_FILENAME = 'MANIFEST.json'
 LOCK_FILE = '/tmp/tmp_lock_file'
```

Comment:
looks like we don't need this anymore, so remove

Reply:
True. I removed this.